src/pages/host/hostDetail/price.jsx

```diff
@@ -0,0 +1,10 @@
+import { useOutletContext } from "react-router-dom";
+import { useParams } from "react-router-dom";
+
+export default function HostPrice() {
```

Comment:
The name of the component needs to match the name of the file.

Reply:
DONE :heavy_check_mark: